### ISSUE TITLE
Generate matplotlibrc params file

### ISSUE THREAD

**Issue by derollins** [AFM-SPM/TopoStats#761]
**Is your feature request related to a problem? Please describe.**
As anticipated in #707 it would be helpful to be able to generate a matplotlibrc params file in a similar way to the config file. It took a while for me to work out how I could change the style of the image output and it currently requires searching around in the code folders to find the file to copy. 
Being able to generate your own would make changing important image formatting options, such as dpi and colour, a lot more user friendly. 

**Describe the solution you'd like**

It would be helpful to be able to quickly generate a matplotlibrc params file that can be easily edited in a similar way to `--create_config_file` .

**Describe alternatives you've considered**
Otherwise the options could be returned to the config file. I certainly change these options fairly regularly while using topostats.

**Comment by ns-rse** [comment]
I have written an entry-point to `topostats create-matplotlibrc` which will generate a copy of the `topostats/topostats.mplstyle` file to a user specified directory and filename (defaults are available).

To avoid merge conflicts I have branched off of branch 757 and so once #758 has been merged I can unblock this issue and create a new pull request to merge this new entry point.
